.github/workflows/test.yml

```diff
@@ -55,9 +56,35 @@ jobs:
           POSTGRES_HOSTNAME: psql-a
           POSTGRES_HOSTNAME_A: psql-a
           POSTGRES_HOSTNAME_B: psql-b
+          
+  # Run unit tests on Linux Swift runners on 
+  linux-unit-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        swiftver:
+          - swift:5.2
+          - swift:5.3
+          - swift:5.4
+          - swiftlang/swift:nightly-5.5
+          - swiftlang/swift:nightly-main
+        swiftos:
+          #- xenial
+          #- bionic
+          - focal
+          #- centos7
+          #- centos8
+          #- amazonlinux2
+    container: ${{ format('{0}-{1}', matrix.swiftver, matrix.swiftos) }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Run tests with Thread Sanitizer
+        run: swift test --enable-test-discovery --sanitize=thread --filter=^PostgresNIOTests
```

Comment:
I wonder if it's worth trying to run these tests in parallel

Reply:
Anything that connects to the database can't run in parallel unless we start having tests generate randomly-named databases (or at least schemas).

Reply:
The PostgresNIOTests don't touch a real DB they're all in the integration tests